pkg/utils/status.go

```diff
@@ -116,6 +116,7 @@ func getDaemonSetStatus(ctx context.Context, instance *csmv1.ContainerStorageMod
 	}
 
 	for _, cluster := range clusterClients {
+		totalRunning = 0
```

Comment:
What does this `totalRunning=0` do? Were we not resetting the count on each iteration of this for loop and needed to be? It doesn't seem related to other code changes in this PR, so it could just be an unrelated fix.